llvm/tools/file-table-tform/file-table-tform.cpp

```diff
@@ -0,0 +1,322 @@
+//===- file-table-tform.cpp - transform files with tables of strings ------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// This tool transforms a series of input file tables into single output file
+// table according to operations passed on the command line. Operations'
```

Comment:
```suggestion
// table according to operations passed on the command line. Operation's
```

Reply:
Operations' = arguments of operations, so English is correct here, as far as I can tell